lib/animina_web/live/chat_live.ex

```diff
@@ -573,7 +573,7 @@ defmodule AniminaWeb.ChatLive do
           </div>
         </.form>
 
-        <%= if messages_sent_to_a_user_by_sender(@sender.id , @messages) == 0 && @current_user_credit_points > 20 && @suggested_messages == [] && @show_use_ai_button do %>
+        <%= if Enum.count(@messages) == 0 && @current_user_credit_points > 20 && @suggested_messages == [] && @show_use_ai_button do %>
```

Comment:
Please don't hardcode those 20 points as a fixed number! Nowhere! Please use a variable for that which gets set in config and can be overridden in dev and prod.

Please fix it for this 20 and search for other occurrences and fix them in an other PR.

BTW: It should be > 19 and not 20.

Reply:
I have set this up and added an issue for it